basics/5 terminal/index.md

```diff
@@ -1,27 +1,147 @@
-# Getting started with Git and the terminal
+# Getting started with Git and the Terminal
+When developing any application, you'll need the Terminal to perform a range of different tasks. You already have experience with this: in the CS50 IDE, the terminal is how you compile and run your code. But as you are getting into more and more advanced methods and tools, you'll start to need it for more and more tasks.
 
-## Opening a terminal application
+So, it is important to be familiar with how the Terminal works. That is what we'll be discussing in this section. We'll do so by introducing a tool you'll often use when working on projects: git.
+
+Don't worry, though. We'll stick to the basics for now. But you can do many awesome things with the terminal alone! More on that in the unit on UNIX, later in the course.
+
+> ### Following along 
+> You'll get the morst out of this section if you follow along with it on your own device. Try out what we're discussing! If you get stuck, or something doesn't quite do what you were expecting, feel free to ask for help!
+
+## Opening a Terminal application
+To do anything with a Terminal, first you need to open a Terminal window. This can be done in many different ways, sometimes depending on your OS and the software you're using.
+
+### Opening a Terminal directly in Visual Studio Code
+Sometimes, you need the Terminal a lot while coding. In that case, it may be best to open a Terminal window directly in Visual Studio Code. 
+
+To do so on a Mac or Linux device, simply navigate to Terminal > New Terminal or hit Ctrl + Shift + ` (should be three keys at the bottom-left of your keyboard).
```

Comment:
Ik heb een beginnetje gemaakt om UNIX vs Windows in side-by-side kolommen te plaatsen. Helaas met lelijke HTML maar dat moet dan maar. Kun je dat bij alle relevante secties doen?

Reply:
Mogelijk moeten de plaatjes daarvoor nog wat geknipt worden, dat kan natuurlijk allemaal. Hoeft niet meteen, eerst even de "big picture". Maar voel je vrij om te klooien tot het er coherent uitziet.

Reply:
PS Let op dat de HTML meestal op aparte regels moet en niet geindenteerd mag zijn!